### DISCUSSION TITLE
Agent Committee Meeting Reminder

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#7411] [Agent Committee]
@ArctosDB/agents-committee this is a reminder that we have a meeting on the calendar for tomorrow, February 14 at 1 PM Pacific in the Arctos Zoom Room.

We plan to demo the new Agent Attribute model and hope to get some feedback from those of you who have worked so hard to make agents in Arctos better!

I hope to see a lot of you there!

**Comment by ewommack** [comment]
I've got a student coming by at that time, so I just need to get them set in the lab and then I will be there.

**Comment by Jegelewicz** [comment]
@ArctosDB/agents-committee this is a reminder that we have a meeting on the calendar for today, March 13 at 1 PM Pacific in the Arctos Zoom Room.

**Comment by Jegelewicz** [comment]
@ArctosDB/agents-committee this is a reminder that we have a meeting on the calendar for Wednesday, April 10 at 1 PM Pacific in the Arctos Zoom Room. I do think we have some things to discuss.

[Agenda](https://docs.google.com/document/d/1-nssmG4VRJFP8Ek1wPDzkfpDoqLMs3eR/edit#heading=h.k5a8oqjsh5a0)

**Comment by Jegelewicz** [comment]
@ArctosDB/agents-committee I think you will all be interested in this - [Workshop Report: Supporting inclusive and sustainable collections-based research infrastructure for systematics (SISRIS)](https://doi.org/10.3897/rio.10.e126532)

**Comment by Jegelewicz** [comment]
A place for us to get appropriate names for terms related to people?

http://xmlns.com/foaf/spec/

Love this:

> The [Person](http://xmlns.com/foaf/spec/#term_Person) class represents people. Something is a [Person](http://xmlns.com/foaf/spec/#term_Person) if it is a person. We don't nitpic about whether they're alive, dead, real, or imaginary. The [Person](http://xmlns.com/foaf/spec/#term_Person) class is a sub-class of the [Agent](http://xmlns.com/foaf/spec/#term_Agent) class, since all people are considered 'agents' in FOAF.

Or from Schema

https://schema.org/Person

> A person (alive, dead, undead, or fictional).
